src/controllers/user/Controller.ts

```diff
@@ -0,0 +1,45 @@
+import { NextFunction, Request, Response } from 'express';
+import { successHandler } from '../../libs/routes';
+import UserRepository from '../../repositories/user/UserRepository';
+
+class UserController {
+
+    public get(req: Request, res: Response) {
+        const user = new UserRepository();
+        const { id } = req.query;
+        user.getUser({ _id: id })
+            .then((data) => {
+                res.status(200).send(successHandler('User Data', 200, data));
+            });
+    }
+
+    public create(req: Request, res: Response) {
+        const { id, name } = req.body;
+        console.log(req.body);
+        const user = new UserRepository();
+        user.createUser({ id, name })
+            .then((data) => {
+                res.status(200).send(successHandler('User Created', 200, data));
+            });
+    }
+
+    public update(req: Request, res: Response) {
```

Comment:
Can you please test this endpoint I think It will give an error since the repo method you called here is not returning a promise.

Reply:
Done